packages/react-native/Libraries/Blob/URL.js

```diff
@@ -104,51 +97,54 @@ export class URL {
       }
       this._url = `${baseUrl}${url}`;
     }
+
+    // Parsing the URL to use for accessors
+    this._parsedUrl = new globalThis.URL(this._url);
```

Comment:
Which JSVM was this tested with?

Reply:
node version: v20.18.1

Reply:
This needs to be tested with Hermes, not Node.

Reply:
okay will run it.

Reply:
v0.13.0